tests/Makefile

```diff
@@ -85,6 +90,19 @@ CLEANING_TARGETS = $(TARGETS) $(FILES_C:%.C=%.o)
 CXXFLAGS += -Wall -std=c++1y -I$(triSYCL_DIR)/include \
 	-I$(triSYCL_DIR)/tests/common
 
+# Vivado HLS path 
```

Comment:
Please configure your editor for not adding trailing spaces or at least to visualize them.